gdal/frmts/nitf/nitffile.c

```diff
@@ -1,2928 +1,3001 @@
-                        szTemp, nTREBytes - 11, nThisTRESize);
-                return NULL;
-            }
-        }
-
-        if( EQUALN(pszTREData,pszTag,6) )
-        {
-            if ( nTreIndex <= 0)
-            {
-                if( pnFoundTRESize != NULL )
-                    *pnFoundTRESize = nThisTRESize;
-
-                return pszTREData + 11;
-            }
-
-            /* Found a previous one - skip it ... */
-            nTreIndex--;
-        }
-
-        nTREBytes -= (nThisTRESize + 11);
-        pszTREData += (nThisTRESize + 11);
-    }
-
-    return NULL;
-}
-
-/************************************************************************/
-/*                        NITFExtractMetadata()                         */
-/************************************************************************/
-
-void NITFExtractMetadata( char ***ppapszMetadata, const char *pachHeader,
-                          int nStart, int nLength, const char *pszName )
-
-{
-    char szWork[400];
-    char* pszWork;
-
-    if( nLength <= 0 )
-        return;
-
-    if (nLength >= (int)(sizeof(szWork) - 1))
-        pszWork = (char*)CPLMalloc(nLength + 1);
-    else
-        pszWork = szWork;
-
-    /* trim white space */
-    while( nLength > 0 && pachHeader[nStart + nLength - 1] == ' ' )
-        nLength--;
-
-    memcpy( pszWork, pachHeader + nStart, nLength );
-    pszWork[nLength] = '\0';
-
-    *ppapszMetadata = CSLSetNameValue( *ppapszMetadata, pszName, pszWork );
-
-    if (szWork != pszWork)
-        CPLFree(pszWork);
-}
-
-/************************************************************************/
-/*        NITF_WGS84_Geocentric_Latitude_To_Geodetic_Latitude()         */
-/*                                                                      */
-/*      The input is a geocentric latitude in degrees.  The output      */
-/*      is a geodetic latitude in degrees.                              */
-/************************************************************************/
-
-/*
- * "The angle L' is called "geocentric latitude" and is defined as the
- * angle between the equatorial plane and the radius from the geocenter.
- *
- * The angle L is called "geodetic latitude" and is defined as the angle
- * between the equatorial plane and the normal to the surface of the
- * ellipsoid.  The word "latitude" usually means geodetic latitude.  This
- * is the basis for most of the maps and charts we use.  The normal to the
- * surface is the direction that a plumb bob would hang were it not for
- * local anomalies in the earth's gravitational field."
- */
-
-double NITF_WGS84_Geocentric_Latitude_To_Geodetic_Latitude( double dfLat )
-
-{
-    /* WGS84 Ellipsoid */
-    const double a = 6378137.0;
-    const double b = 6356752.3142;
-
-    /* convert to radians */
-    dfLat = dfLat * M_PI / 180.0;
-
-    /* convert to geodetic */
-    dfLat = atan( ((a*a)/(b*b)) * tan(dfLat) );
-
-    /* convert back to degrees */
-  
```

Comment:
I suspect there must be some newline changes, or perhaps just the change of the file permissions, that cause the difference to appear to be on the whole file and makes it unreviewable